File: Sakila_Net/Sakila_Net/CS_SakilaLib/Neg/NegCategory.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegCategory
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOCategory.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClCategory category) 
        {
            return DAO.DAOCategory.sqlSelect(category);
        }
        public  static int crear(ClCategory category)
        {
            return DAO.DAOCategory.sqlInsert(category);
        }
        public  static int eliminar(ClCategory category)
        {
            return DAO.DAOCategory.sqlDelete(category);
        }
        public  static int actualizar(ClCategory category)
        {
            return DAO.DAOCategory.sqlUpdate(category);
        }
    }
}

File: Sakila_Net/VB_SakilaLib/Neg/NegCategory.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegCategory
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOCategory.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( category as ClCategory) as Boolean
                        Return DAO.DAOCategory.SelectOne(category)     
                    End Function
                    Public  Shared Function crear( category as ClCategory) as Integer
                        Return DAO.DAOCategory.SqlInsert(category)     
                    End Function
                    Public  Shared Function eliminar( category as ClCategory) as Integer
                        Return DAO.DAOCategory.SqlDelete(category)     
                    End Function
                    Public  Shared Function actualizar( category as ClCategory) as Integer
                        Return DAO.DAOCategory.SqlUpdate(category)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  
